packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -159,6 +159,7 @@ export class TspLanguageClient {
         logger.error(
           [
             `TypeSpec server executable was not found: '${this.exe.command}' is not found. Make sure either:`,
+            " - Node.js is installed locally and available in PATH.",
```

Comment:
i think we can know the command is "node", please update to only add the line when the command is "node". thx